src/components/Global/BaseSnackbar.vue

```diff
@@ -1,6 +1,12 @@
 <template>
+  <!--
+    Using v-model is not allowed, in a base component such as this we need to adhere
+    to the one-way data flow Vue prefers and take the value, then emit it back up manually
+    see https://vuejs.org/guide/components/props.html#one-way-data-flow
+  -->
   <v-snackbar
-    v-model="modelValue"
+    :modelValue="modelValue"
+    @update:modelValue="modelValue"
```

Comment:
Shouldn't this emit the value up? It looks like you're treating a boolean variable like it were a function by passing it as a handler here
```suggestion
    @update:modelValue="$emit(modelValue)"
```

Reply:
Yep great catch, thanks.